python/rgbmatrix/core.pyx

```diff
@@ -83,18 +83,100 @@ cdef class FrameCanvas(Canvas):
         def __get__(self): return (<cppinc.FrameCanvas*>self.__getCanvas()).pwmbits()
         def __set__(self, pwmBits): (<cppinc.FrameCanvas*>self.__getCanvas()).SetPWMBits(pwmBits)
 
+cdef class RGBMatrixOptions:
+    def __cinit__(self):
```

Comment:
There should be either the constructor or some InitFromFlags() that takes argc and argv and then calls the underlying rgb_matrix::ParseOptionsFromFlags().

Best: If we have global access to the raw argv/argc in the constructor, we should probably do that there, so that the defaults are automatically set from what the user gave on the command line.

This way, options don't have to be manually added to the samplebase.py and things are always consistent with the C++ demos, otherwise this is a maintenance nightmare, because the C++ library changes relatively often and the Python lib only whenever someone knowledgeable in Python contributes a patch.